### DISCUSSION TITLE
Hosted edition?

### DISCUSSION THREAD

**Discussion by ottok** [umputun/remark42#1748] [Ideas]
Do you have any plans to offer a hosted edition?

[Cusdis](https://cusdis.com/) is a another similar system that is open source, but also offers a hosted plan. I would prefer using Remark42 however.

**Comment by umputun** [comment]
moved to discussions

**Comment by colans** [comment]
This is really the only thing blocking me from using it right now.  I don't want to take the time to set up and maintain it so I'm going to run with Disqus for now, and migrate later, when this is available.

**Reply by colans** [reply]
Actually, if someone is willing to pay me for the work, I'd be happy to set up a SaaS version.

**Comment by paskal** [comment]
It would be great to have a hosted edition, but there are a few obstacles in the current implementation preventing that, particularly the lack of multi-domain support in the authentication library. Someday, I plan to fix this, and then we’ll be closer to offering a hosted edition. However, for now, self-hosting is unfortunately the only option.

**Reply by colans** [reply]
Sounds like a better idea than doing a bunch of single tenant instances.

**Reply by mnordhaus** [reply]
Well, you might be able to run multiple Remark42 instances on one machine. Then they are completely separated and still you won't have to pay an ideling  EC2, just to have one website served with comments ... Does that sound like an idea?

**Reply by umputun** [reply]
Well, how big do you really need? This is not a heavy real-time system, and all the hot data from BoltDB is aggressively cached. I know about several installations on big sites with massive traffic, and from what I understand, they were able to use Remark42 as is, without any modifications, at least not performance-related modifications.

upd: replied to a comment claiming that boltdb is the bottleneck for high-load sites. The comment is gone (removed?)
